spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -1090,34 +1090,63 @@ The lifecycle of a schedule has 2 parts:
 
 You can schedule a task execution via the:
 
-* RESTful API
+* Spring Cloud Data Flow Shell
 * Spring Cloud Data Flow Dashboard
+* Spring Cloud Data Flow RESTful API
 
-To schedule a task from the UI click the Tasks tab at the top of the screen, this will take you to the Task Definitions screen.  From the Task Definition that you wish to schedule, please click the dropdown icon and then select the `Schedule Task` option.
-This will lead you to a `Create Schedule(s)` screen, where you will create a unique name for the schedule and enter the associated cron expression.
-Keep in mind you can always create multiple schedules for a single task definition.
+==== Scheduling a Task
+To schedule a task using the shell, use the `task schedule create` command to create the schedule as shown in the following example:
+```
+dataflow:>task schedule create --definitionName mytask --name mytaskschedule --expression '*/1 * * * *'
+Created schedule 'mytaskschedule'
+```
+In the example above we created a schedule `mytaskschedule` for the task definition `mytask`.  This schedule will launch `mytask` once a minute.
+
+NOTE: When using Cloud Foundry the cron expression will be: `*/1 * ? * *`.
```

Comment:
@cppwfs Good that you pointed this out :-) Could you also specify why this is required?